content/applications/finance/accounting/reporting/overview/customize.rst

```diff
@@ -1,90 +1,248 @@
+A formula made for this engine consists of a name used to match tax tags. If such tags do not exist
+when creating the expression, they will be created.
 
-.. seealso::
-    * :doc:`main_reports`
+.. example::
+   If the formula is **tag_name**, the engine matches tax tags **+tag_name** and **-tag_name**,
+   creating them if necessary.
+
+When evaluating the expression, the expression computation can roughly be expressed as: **(amount of
+the move lines with** `+` **tag)** `-` **(amount of the move lines with** `-` **tag)**.
+
+'Aggregate Other Formulas' engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use this engine when you need to perform arithmetic operations on the amounts obtained for other
+expressions. Formula can be any expression with `+`, `-`, `/` and `*` operator. To refer to an
+expression, use the code of its **parent line** and its **label** (ex. **CODE.label**).
+
+**Subformulas** can be one of the following:
+
+- **if_above(CUR(amount))**: The value of the arithmetic expression will be returned only if it's
+  greater than the provided bound. Else, the result will be `0`;
+
+- **if_below(CUR(amount))**: The value of the arithmetic expression will be returned only if it's
+  lower than the provided bound. Else, the result will be `0`;
+
+- **if_between(CUR1(amount1), CUR2(amount2))**: The value of the arithmetic expression will be
+  returned only if it's strictly between the provided bounds. Else, it will be brought back to the
+  closest bound.
+
+`CUR` is the currency code in capital letters, and `amount` is the amount of the bound expressed in
+that currency.
+
+You can also use the `cross_report` subformula to match, if your expression refers to an expression
+defined in another report.
+
+'Prefix of Account Codes' engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This engine is used to match amounts made on accounts using the prefixes of these accounts' codes as
+variables in an arithmetic expression.
+
+.. example::
+   | **21 + 10 - 5**
+   | Adds the balances of the move lines made on accounts whose codes start with `21` and `10`, and
+     subtract the balance of the ones on the accounts with the prefix `5`.
+
+It is also possible to ignore a selection of sub-prefixes.
+
+.. example::
+   | **21 + 10\\(101, 102) - 5\\(57)**
+   | Does the same as the previous example, but ignoring the prefixes `101`, `102` and `57`.
+
+You can apply 'sub-filtering' on **credits and debits** using `C` and `D` suffixes. In this case, an
+account will only be considered if its prefix matches, *and* if the total balance of the move lines
+made on this account is **credit/debit**.
+
+.. example::
+   Account `210001` has a balance of -42 and account `210002` has a balance of 25. The formula
+   **21D** only matches the account `210002`, and hence return 25. `210001` is not matched, as its
+   balance is *credit*.
+   
+Prefix exclusions can be mixed with `C` and `D` suffixes.
+
+.. example::
+   **21D + 10\\(101, 102)C - 5\\(57)**
```

Comment:
@oco-odoo

Reply:
To me, an additional explanation is not necessary, since it's just combining the previous examples.

But if you really want one, here it is:

This example will include the balance of the accounts whose code starts with 21 if it's debit (so, if it's positive), the balance of the accounts starting with 10 if it's credit, excepting prefixes 101 and 102, and the total balance of all accounts starting with 5, except 57. 

Not sure how to phrase it in a clear and not too verbose way :/